cdc/sink/mysql.go

```diff
@@ -511,26 +524,19 @@ func newMySQLSink(
 	if params.timezone != "" {
 		dsn.Params["time_zone"] = params.timezone
 	}
-	testDB, err := sql.Open("mysql", dsn.FormatDSN())
+	testDB, err := getDBConnImpl(ctx, dsn.FormatDSN())
 	if err != nil {
-		return nil, errors.Annotate(
-			cerror.WrapError(cerror.ErrMySQLConnectionError, err), "fail to open MySQL connection when configuring sink")
+		return nil, err
```

Comment:
```suggestion
		return nil, errors.Trace(err)
```

Reply:
err has contained error trace, `errors.Trace` is not necessary here

Reply:
Yes, though I think it would idiomatic to wrap errors as suggests from wrapcheck linter https://golangci-lint.run/usage/linters/#:~:text=wrapcheck

Reply:
Maybe not suitable for the current error mechanism. Currently all TiCDC errors should be under the TiDB error RFC, which means
- error from an external package should be wrapped by an RFC error generator
- error generated by TiCDC internal logic must be generated from an RFC error generator